events/HF_battle_events.txt

```diff
@@ -2179,11 +2223,6 @@ character_event = {
 	is_triggered_only = yes
 	
 	immediate = {
-		random = {  #Quick fix
```

Comment:
idk why you changed that

Reply:
Is this a Zumbak fix?

Reply:
Probably, since there no such thing in vanilla code

Reply:
@zumbak04

Reply:
No idea what is it and who added it, so I purged it.